index.html

```diff
@@ -29,7 +34,7 @@
     <section class="header-page">  
       <div class="header-container">
           <div class="headline">
-            <h1>I’m Emmanuel Glad to see <br/> you!</h1>
+            <h1>I’m Emmanuel Glad to see you!</h1>
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
             <div class="social-media">
```

Comment:
Please use the `ul` tag as a container for social icons and every icon should be wrapped inside a `li` tag.